pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -302,15 +302,31 @@ def select_channel(self, channel_name):
         """
         self.write(f"INST:SEL '{channel_name}'")
 
-    # active_channel = Instrument.control("INST?", "INST %s", "To get the active channel")
+    @property
+    def active_channel(self):
+        return self.values("INST?")[0]
+
+    @active_channel.setter
+    def activate_channel(self, channel):
+        availabel_channels = [chan for chan in self.available_channels.keys()]
+        channel = strict_discrete_set(channel, availabel_channels)
+        self.write(f"INST '{channel}'")
+
+    def view_mode(self, view="SPL"):
+        """To get a MultiView of the open channels or return to the last active channel
+        Args:
+            view (str, optional): To view MultiView. Defaults to "SPL". To go back to the last
```

Comment:
Did not check the manual but it look like `SPL` split means "split (view)" and `SING` means "single (view)". So I think it makes more sense to use a property `split_view` that accepts a boolean that states whether the view is split or not.

Should then be turned into a property, probably via the `Instrument.control` factory.